pages/give/endowments/[slug].vue

```diff
@@ -0,0 +1,325 @@
+<script setup>
+// HELPERS
+import _get from 'lodash/get'
+import removeTags from '../utils/removeTags'
+
+// GQL
+import ENDOWMENT_DETAIL from '../gql/queries/EndowmentDetail.gql'
+
+const { $graphql } = useNuxtApp()
+
+const route = useRoute()
+
+const { data, error } = await useAsyncData('endowment-detail', async () => {
```

Comment:
use a unique key like `endowment-detail-${route.params.slug}`, please make this change on location detail, news detail, and other slug pages. a unique key to ensure that data fetching can be properly de-duplicated across requests.